ddemangle.d

```diff
@@ -85,20 +109,41 @@ unittest
         "fail demangling __D6object9Throwable8toStringMFZAy"
     ];
 
-    assert(equal(testData.map!ddemangle(), expected));
+    assert(equal(testData.map!(a=>a.ddemangle(false)), expected));
```

Comment:
style: spacing around `=>`

Reply:
done (had added that after running dfmt)